cypress/integration/pro_deputy_hub.spec.js

```diff
@@ -91,6 +91,28 @@ describe("Pro Deputy Hub", () => {
         });
     });
 
+    describe("Important information", () => {
```

Comment:
It doesn't matter hugely because this is a small test suite and runs quickly but best practice with Cypress is have fewer tests that test more information, in order to avoid reloading the state for every test.

Instead of having multiple `it` blocks, it might be better to have one that checks all these important information attributes. If you want to keep them named nicely, you could assign the elements to named variables.